lib/src/featureTypes/multi_polygon.dart

```diff
@@ -95,16 +95,15 @@ class MultiPolygon extends Feature {
   /// ```
   @override
   factory MultiPolygon.fromWKT(String wkt) {
-    final polygonType = wkt.matchAsPrefix('MULTIPOLYGON');
-    if (polygonType == null) {
+    final polygonType = wkt.startsWith('MULTIPOLYGON');
+    if (!polygonType) {
       throw ArgumentError('wkt is not a MultiPolygon');
     }
```

Comment:
I think this startsWith check should be added to Polygon.fromWKT too for consistency.

Reply:
Done!